src/frameworks/hibernate.md

```diff
@@ -0,0 +1,97 @@
+# Hibernate
+
+[Hibernate ORM](https://hibernate.org/) is an object-relational mapping tool for the Java programming language. It provides a framework for mapping an object-oriented domain model to a relational database. Hibernate handles object-relational impedance mismatch problems by replacing direct, persistent database accesses with high-level object handling functions.
+
+## Services
+
+The [configuration reader library](https://github.com/platformsh/config-reader-java) for Java is used in these examples, so be sure to check out the [documentation](/languages/java.html#support-libraries) for installation instructions and the latest version.
+
+### MySQL
+
+[MySQL](configuration/services/mysql.md) is an open-source relational database technology. Define the driver for [MySQL](https://mvnrepository.com/artifact/mysql/mysql-connector-java), and the Java dependencies. Then determine the SessionFactory client programmatically:
```

Comment:
The internal link here needs to begin with a /, or else it will be relative to the current page, which is wrong.

```suggestion
[MySQL](/configuration/services/mysql.md) is an open-source relational database technology. Define the [driver for MySQL](https://mvnrepository.com/artifact/mysql/mysql-connector-java), and the Java dependencies. Then determine the SessionFactory client programmatically:
```